payments/credit-card/redirect.md

```diff
@@ -151,55 +154,55 @@ Content-Type: application/json
+|  ✔︎︎︎︎︎  | currency                                  | string      | NOK, SEK, DKK, USD or EUR.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+|  ✔︎︎︎︎︎  | prices.type                               | string      | Use the generic type CreditCard if you want to enable all card brands supported by merchant contract. Use card brands like Visa (for card type Visa), MasterCard (for card type Mastercard) and others if you want to specify different amount for each card brand. If you want to use more than one amount you must have one instance in the prices node for each card brand. You will not be allowed to both specify card brands and CreditCard at the same time in this field. [See the Prices resource and prices object types for more information][price-resource].                                         |
+|  ✔︎︎︎︎︎  | prices.amount                             | integer     | Amount is entered in the lowest monetary units of the selected currency. E.g. 10000 = 100.00 NOK, 5000 = 50.00 SEK.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|  ✔︎︎︎︎︎  | prices.vatAmount                          | integer     | If the amount given includes VAT, this may be displayed for the user in the payment page (redirect only). Set to 0 (zero) if this is not relevant.                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
`prices.vatAmount` should be replaced with the indent arrow used elsewhere and `vatAmount` without the `prices.` prefix (which should be described separately).